notary-server/Dockerfile

```diff
@@ -31,7 +7,7 @@ ENV INSTALLDIR /notary/server
 ENV PATH=$PATH:${INSTALLDIR}
 WORKDIR ${INSTALLDIR}
 
-COPY --from=build /notary-server ./
+COPY --from=notary:0.7.0-builder /notary-server ./
```

Comment:
Just to document why the builder image with build tools and both binaries is tagged as `0.7.0-builder` instead of `builder-0.7.0`.

Current image tags are `server-0.7.0` and `signer-0.7.0` (and aliased without version). I chose `[version]-builder` over `builder-[version]` because the builder image is not a specific Notary binary like `server` and `signer` are and is mostly an implementation detail of how the others are built.

Happy to change my opinion if the Notary maintainers would rather use `builder-0.7.0`.